plugins/woocommerce/templates/global/quantity-input.php

```diff
@@ -12,7 +12,7 @@
  *
  * @see     https://woocommerce.com/document/template-structure/
  * @package WooCommerce\Templates
- * @version 7.8.0
+ * @version 7.8.1
```

Comment:
Version should be that of next release, so 9.3.0
```suggestion
 * @version 9.3.0
```

Reply:
Thank you! :)